models/surveys.go

```diff
@@ -1,18 +1,23 @@
 package models
 
+// TODO All transactions should have rollbacks in the case of errors
```

Comment:
Is there any reason why these TODOs are not just done as part of this PR, as opposed to adding tech debt?

Reply:
It's a good question :) It's not technical debt added by this PR (all the bits touched by the changes here have had that done). It was more an observation on the existing code that needs to be fixed at some point (outside the scope of this change). 
A card has been raised in the backlog to look at this.

Reply:
ah ok, no boy scout rule here then.